project/profiles/capacity4more/modules/c4m/message/c4m_message/c4m_message.module

```diff
@@ -77,6 +99,173 @@ function c4m_message_entity_load($entities, $entity_type) {
 }
 
 /**
+ * Add callbacks to c4m_welcome_to_group message arguments.
+ *
+ * @param \Message $message
+ *
+ */
+function c4m_message_load__c4m_welcome_to_group(Message $message) {
+  $message->arguments['@member_first_name'] = array(
+    'callback' => 'c4m_message_format_member_first_name',
+    'pass message' => TRUE,
+  );
+
+  $message->arguments['@member_last_name'] = array(
+    'callback' => 'c4m_message_format_member_last_name',
+    'pass message' => TRUE,
+  );
+
+  $message->arguments['!group_dashboard_url'] = array(
+    'callback' => 'c4m_message_format_group_path',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['@group_node_title'] = array(
+    'callback' => 'c4m_message_format_group_node_title',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['@group_welcome_message'] = array(
+    'callback' => 'c4m_message_format_group_welcome_message',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['@group_node_type'] = array(
+    'callback' => 'c4m_message_format_group_node_type',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['!group_dashboard_url'] = array(
+    'callback' => 'c4m_message_format_group_url',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['!go_public_profile_url'] = array(
+    'callback' => 'c4m_message_format_go_public_profile_url',
+    'pass message' => TRUE,
+  );
+
+  //Add
+  $message->arguments['@go_full_name'] = array(
+    'callback' => 'c4m_message_format_go_full_name',
+    'pass message' => TRUE,
+  );
+}
+
+/**
+ * Return a member's first name.
+ *
+ * @param \Message $message
+ * @return string
+ */
+function c4m_message_format_member_first_name(Message $message) {
```

Comment:
Remove this function and the one of the __last name__ and add one to handle the __full name__ instead.